recipes/methpype/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: methpype
+  version: 1.0.1
+
+source:
+  url: https://files.pythonhosted.org/packages/00/88/eab68b60ad237ba0f9b69f862b6b92553a9646b4e5620a06e58803fa645d/methpype-1.0.1.tar.gz
+  sha256: acf192a02388a2eb71fe4ae6d0bac13b174c53fb3bb98f95eaaa9e17b2394597
+
+build:  
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - methpype
+
+about:
+  home: http://github.com/LifeEGX/methpype
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
The ``license`` is not being packaged on pypi, you will have to add it manually then
Please see
https://conda-forge.readthedocs.io/en/latest/meta.html#packaging-the-license-manually